libibverbs/man/ibv_poll_cq.3

```diff
@@ -83,7 +83,8 @@ overrun from occurrence.  In case of a CQ overrun, the async event
 .B IBV_EVENT_CQ_ERR
 will be triggered, and the CQ cannot be used.
 .PP
-IBV_WC_DRIVER1 will be reported as a response to IBV_WR_DRIVER1 opcode.
+IBV_WC_DRIVER1 will be reported as a response to IBV_WR_DRIVER1 opcode;
+IBV_WC_DRIVER2 will be reported on a specific driver operation.
```

Comment:
What does that mean? Why isn't IBV_WC_DRIVER2 used with IBV_WR_DRIVER2?

Reply:
It points on a driver specific completion opcode with that value (i.e. IBV_WR_DRIVER2).

For mlx5 it will be used with raw WQE builder, see its man page, any driver can use it along with its specific usage.

There was no need / usage for IBV_WR_DRIVER2 and as such it wasn't added as part of this series.